### ISSUE TITLE
Many errors occurred at compile time

### ISSUE THREAD

**Issue by ghost** [KiraYamato94/LasMonjas#4]
**I found a lot of errors after compiling this program, I don't know how these errors are generated, I went to Github to download LasMonjas source code again, but the problem could not be solved**
![image](https://user-images.githubusercontent.com/106727385/179152068-a6d811bb-322f-48a0-83d0-aeb0a3879e11.png)
This is Chinese, it mean **You have 1 warning, 1971 errors**

I downloaded different versions of [Dotnet](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) in a row and couldn't solve it

do you konw why? If you have time, you can send me your Dotnet installer, thanks

**Comment by KiraYamato94** [comment]
I'll upload 2.1.0 later, try to compile it and if it works, start translating from there. I'm no longer active-development this mod

**Comment by ghost** [comment]
Let me try again. Let me make it Chinese. But what kind of dotnet would be better? What if I try to use a different version of Dotnet and he can't compile?
